SonarQube.Bootstrapper/BootstrapperSettings.cs

```diff
@@ -149,23 +155,22 @@ public int PostProcessorTimeoutInMs
 
         #region Private methods
 
-        private static string CalculateDownloadDir(Settings settings, ILogger logger)
+        private static string CalculateTempDir(Settings settings, ILogger logger)
         {
-            // Use the app setting if supplied...
-            string dir = settings.DownloadDir;
-            if (string.IsNullOrWhiteSpace(dir))
-            {
-                // ... otherwise work it out from the environment variables
-                logger.LogMessage(Resources.INFO_UsingEnvVarToGetDirectory);
-                string rootDir = GetFirstEnvironmentVariable(DirectoryEnvVarNames, logger);
+            logger.LogMessage(Resources.INFO_UsingEnvVarToGetDirectory);
+            string rootDir = GetFirstEnvironmentVariable(DirectoryEnvVarNames, logger);
 
-                if (!string.IsNullOrWhiteSpace(rootDir))
-                {
-                    dir = Path.Combine(rootDir, RelativePathToDownloadDir);
-                }
+            if (string.IsNullOrWhiteSpace(rootDir))
+            {
+                rootDir = Directory.GetCurrentDirectory();
             }
 
-            return dir;
+            return Path.Combine(rootDir, RelativePathToTempDir);
+        }
+
+        private static string CalculateDownloadDir(Settings settings, ILogger logger)
```

Comment:
Is this still used? And if it is, consider using the TempDirectory getter instead of recomputing the temp dir